File: messaging/v1/short_code/read-service-shortcode-list/read-service-shortcode-list.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

short_codes = @client.messaging
                     .services('MG2172dd2db502e20dd981ef0d67850e1a')
                     .short_codes
                     .list(limit: 20)

short_codes.each do |record|
  puts record.sid
end

File: messaging/v1/short_code/read-service-shortcode-list/read-service-shortcode-list.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$shortCodes = $twilio->messaging->v1->services("MG2172dd2db502e20dd981ef0d67850e1a")
                                    ->shortCodes
                                    ->read();

foreach ($shortCodes as $record) {
    print($record->sid);
}